playbooks/incident-handling.md

```diff
@@ -0,0 +1,458 @@
+   ([examples](https://twitter.com/stripestatus)). Technical details or system code names are not relevant or
+   appropriate.
+
+### 3. Address
+
+#### Roles
+
+The first on-call engineer on the scene should
+[take the role of **Incident Commander**](<(https://docs.opsgenie.com/docs/incident-response-roles)>) unless
+there's another volunteer or naturally responsible candidate. Your responsibility in this role is to _facilitate_ a
+successful resolution, not to solve the problem yourself.
+
+If both on-call engineers are available, the 2nd should take the role of **Communications Officer**, handling
+updates to internal stakeholders (on the #incidents thread) and external audiences (on the status page), as well as
+recording relevant information (insights, links...) on the incident timeline. If both engineers aren't available,
+the incident commander can perform both roles or ask for another volunteer.
+
+#### Diagnose
+
+Discuss the incident's resolution in the dedicated Slack channel. Tips:
+
+- Scan [recent releases](https://github.com/pulls?q=is%3Amerged+is%3Apr+archived%3Afalse+org%3Aartsy+deploy) that
+  may be responsible.
+  [Roll back](https://github.com/artsy/hokusai/blob/master/docs/Command_Reference.md#how-to-do-a-rollback) the
+  culprit if necessary.
+- Start a video conference to communicate in real-time with other responders (see 1Password for shared Zoom account
+  credentials).
+- See
+  [Engineering Playbooks (Notion)](https://www.notion.so/artsy/Engineering-Playbooks-b655fe54c1ce4b35af342c9ed9a489ae)🔒
+  and [legacy wiki docs](https://github.com/artsy/potential/wiki)🔒 for solutions to common problems.
+- Datadog [monitors](https://app.datadoghq.com/monitors/triggered?q=),
+  [service traces](https://app.datadoghq.com/apm/services), and
+  [dashboards](https://app.datadoghq.com/dashboard/lists) can illuminate the source and scale of latency or errors.
+- Error details are [captured in Sentry](https://sentry.io/organizations/artsynet/issues).
+- Logs are searchable [in Papertrail](https://papertrailapp.com/dashboard).
+- The [Kubernetes dashboard](https://kubernetes.prd.artsy.systems/) reflects the configuration, scale, or events
+  relating to specific system deployments.
+- See also other metrics and dashboards (AWS, MongoDB, Elasticsearch/kibana, Sendgrid...).
+
+If a mitigation (such as rolling back a release) requires some follow-up, make sure to note it on the incident and
+surface to the appropriate team. If a mitigation isn't immediately apparent, **escalate to an appropriate team or
+engineer**.
+
+#### Escalate
+
+When investigating an incident, you should involve the relevent team and stakeholders involved as soon as possible.
+**You are not expected to know the ins and outs of every system, so don't hesitate to involve the wider team.**
+
+<details>
+  <summary>To escalate an issue:</summary>
+
+- Identify the engineer responsible for the associated pull request or release. If not obvious, the
```

Comment:
We should add a line here about the first line of escalation being the Team x Scope list-- i.e. if there's an issue with payments, we want the person to contact the Transact team first, not necessarily the Exchange point-people.

Reply:
Let me know what you think of the updates.